agent/pkg/chassis/protocol/http/http.go

```diff
@@ -52,19 +52,9 @@ func (p *HTTP) Process() {
 	for {
 		// parse http request
 		req, err := http.ReadRequest(bufio.NewReader(p.Conn))
-		if err != nil {
-			if err == io.EOF {
-				klog.Warningf("read http request EOF")
-				err = p.Conn.Close()
-				if err != nil {
-					klog.Errorf("close conn err: %v", err)
-				}
-				return
-			}
-			klog.Errorf("read http request err: %v", err)
-			err = p.Conn.Close()
-			if err != nil {
-				klog.Errorf("close conn err: %v", err)
+		if err != nil || err == io.EOF {
+			if err = p.Conn.Close(); err != nil {
+				klog.V(4).ErrorS(err, "close conn error")
```

Comment:
We can just print the error out
```suggestion
				klog.Errorf("close conn err: %v", err)
```

Reply:
fix